cardpresent/src/main/java/com/fattmerchant/android/Mock/MockDriver.kt

```diff
@@ -0,0 +1,161 @@
+package com.fattmerchant.android.Mock
+
+import com.fattmerchant.android.chipdna.ChipDnaDriver
+import com.fattmerchant.omni.MobileReaderConnectionStatusListener
+import com.fattmerchant.omni.SignatureProviding
+import com.fattmerchant.omni.TransactionUpdateListener
+import com.fattmerchant.omni.UserNotificationListener
+import com.fattmerchant.omni.data.*
+import com.fattmerchant.omni.data.MobileReaderDriver
+import com.fattmerchant.omni.data.models.MobileReaderDetails
+import com.fattmerchant.omni.data.models.OmniException
+import com.fattmerchant.omni.data.models.Transaction
+
+internal fun createMockReader(forIndex: Int): MobileReader {
+    return object : MobileReader {
+        override fun getName() = "Reader $forIndex"
+        override fun getFirmwareVersion() = "FakeFirmwareVersion$forIndex"
+        override fun getMake() = "FakeMakeR$forIndex"
+        override fun getModel() = "FakeModelR$forIndex"
+        override fun serialNumber() = "FakeSerialNumber$forIndex"
+    }
+}
+
+internal final class MockDriver: MobileReaderDriver {
+
+    override val source: String = "MOCKSOURCE"
+    override var familiarSerialNumbers: MutableList<String> = mutableListOf()
+    override var mobileReaderConnectionStatusListener: MobileReaderConnectionStatusListener? = null
+
+    /// Set this to false to simulate a busy mobile reader
+    var readyToTakePayment: Boolean = true
+
+    var isInitialized: Boolean = true
+    var shouldConnect: Boolean = true
+
+    var currentlyConnectedReader: MobileReader? = null
+
+    override suspend fun isReadyToTakePayment(): Boolean {
+        return readyToTakePayment
+    }
+
+    override suspend fun isOmniRefundsSupported(): Boolean {
+        return false
+    }
+
+    override suspend fun initialize(args: Map<String, Any>): Boolean {
+        val nmiDetails = args["nmi"] as? MobileReaderDetails.NMIDetails
```

Comment:
try to use static import   **NMIDetails**

Reply:
Addressed in d57929f71e8dff91dd42a4e74d7e4a1198bea3d5